cmd/snap/cmd_reseal.go

```diff
@@ -0,0 +1,66 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/i18n"
+)
+
+var (
+	shortResealHelp = i18n.G("Reseal this device")
+	longResealHelp  = i18n.G(`
+TODO
+`)
+)
+
+type cmdReseal struct {
+	waitMixin
+}
+
+func init() {
+	addCommand("reseal",
```

Comment:
as it's related to booting/system I would perhaps make it a subcommand of "snap reboot":

snap reboot --reseal

we have a few subcommands attached that way to other commands

Reply:
I have not yet removed this command. I have added  `snap reboot --reseal` however.